ci/ext/post_package.d/kabanero_update_index.sh

```diff
@@ -26,4 +26,10 @@ fi
 
 NGINX_IMAGE=nginx-ubi
 image_build -t $NGINX_IMAGE \
- -f $script_dir/nginx-ubi/Dockerfile $script_dir
+ -f $script_dir/nginx-ubi/Dockerfile $script_dir \
+> ${build_dir}/image.$NGINX_IMAGE.log
+if [ "${VERBOSE}" == "true" ]; then
```

Comment:
Are these if checks necessary? Given trace already checks for VERBOSE then we shouldn't need to check here.

Reply:
Or should this be a check for failure and do the echo regardless of verboseness if the build fails?